conf/radon.default.json

```diff
@@ -12,5 +13,8 @@
         },
         "log": {
                 "level": "INFO"
+        },
+        "monitor": {
+            "monitor-address": "0.0.0.0:13308"
```

Comment:
monitor port should suffix with 80, *3308 is mysql port in Radon, here maybe confuse for users.

Reply:
OK, I changed the port "13308" to "13380".